scripts/gceworker.sh

```diff
@@ -19,6 +19,11 @@ fi
 
 function start_and_wait() {
     gcloud compute instances start "${1}"
+    if [ -z "${GCEWORKER_NO_FIREWALL_UPDATE-}" ]; then
+	# https://cockroachlabs.atlassian.net/wiki/spaces/devinf/pages/65077811/Building+CRDB+with+gceworker#Add-Your-IP-to-the-GCE-SSH-Allow-List
+        echo "updating firewall rule (disable with GCEWORKER_NO_FIREWALL_UPDATE=true)..."
+        $0 update-firewall
```

Comment:
Nit: preference to put this in a `function` rather than re-call the script with the `update-firewall` argument